scheduler/controllers/clustercollectorsgroup/resource_config.go

```diff
@@ -6,8 +6,19 @@ import (
 )
 
 const (
+	// the default memory request in MiB
 	defaultRequestMemoryMiB = 500
 
+	// the default CPU request in millicores
+	defaultRequestCPUm = 500
+	// the default CPU limit in millicores
+	defaultLimitCPUm = 1000
+
+	// MinReplicasDefault is the default number of replicas for the collector
+	MinReplicasDefault = 1
+	// MaxReplicasDefault is the default maximum number of replicas for the collector hpa
+	MaxReplicasDefault = 10
```

Comment:
Above we have `default***` should we use the same name convention here as well?
Also, should it start with Capital letter?